docs/watch_tutorial/index.md

```diff
@@ -0,0 +1,362 @@
+
+In this guide we'll be building a simple digital watch; we'll utilize panels and 3D models and implement a a little interactivity. Once you've worked through this guide, you should have two things:
+
+* A vintage, functioning 90s G-Shock watch gadget 
+* A better understanding of the Aardvark scene graph, familiarity with useful nodes, and the gadget development workflow.
+
+# Step 1: _The First Time_ - Tooling and debugging
+
+If you've worked on wearable VR/AR experiences, you're already familiar with the ergonomic difficulties of iterating on development and alternating between coding (headset off) and testing (headset on). You might already have a workflow you're comfortable with that mitigates this, in which case feel free to skim or skip this section. In any case, here's my typical workflow in the hope that it might be of use to you too:
+
+1. I like to stay in VR and work in the virtual desktop to minimize headset fidgeting time. It's worth knowing there are several good VR desktop solutions available like [this one, ](https://www.vrdesktop.net/) [this one, ](https://store.steampowered.com/app/457550/Bigscreen_Beta/) [this one](https://store.steampowered.com/app/1068820/OVR_Toolkit/) and [this one](https://store.steampowered.com/app/1178460/Desktop_Portal/) that play well with SteamVR, some of which look and feel much better than the built-in one. 
+
+2. Coding is done in VS Code, with the `npm start` script running in VS Code's terminal (you can launch a new terminal instance from VS Code's Terminal tab):
+   
+![workflow-1](./media/0-0-workflow.jpg)
+
+Note that the script watches the files and recompiles on change, so you don't have to "refresh" anything aside from hitting save.
+
+2. When iterating, the Aardvark renderer does *not* have to be restarted to reflect gadget updates but you *do* have to instantiate a new gadget every time you want to see the latest compiled changes (and probably want to discard that gadget after testing).
+This is what my workflow typically looks like - IDE, test in SteamVR, back to IDE:
+
+![workflow.gif](./media/workflow.gif)
+
+3. For debugging, remember you can use Chrome's devtools on the Aardvark Server (http://localhost:8042/), as well as the Gadget Monitor (http://localhost:23842/gadgets/aardvark_monitor/index.html). You can also debug your gadget from VS Code, provided you've installed the Debugger for Chrome extension and answered "yes" to the `Do you want to debug with VS Code?` question when [instantiating your gadget](https://aardvarkxr.github.io/aardvark/getting_started/). 
+
+
+# Step 2: _Time of the Season_ - Adding a time/date panel
+
+Alright, now that we've got a dev environment and a nice workflow, we can jump into the code. 
+
```

Comment:
Assuming CORS cooperates you could also host the media files in your repo (turn on GitHub Pages for the whole repo) and use absolute urls to refer the the assets.

Reply:
yeah I think hosting them on another repo doesn't make sense. 
I think having the PROJECT_ASSETS folder live in the same folder as `index.md` makes the most sense (so working through the tut still means copying to the new gadget's `src` folder, modifying the webpack config file, etc. - make it less plug and play.)